slack-dark-mode.sh

```diff
@@ -39,23 +39,29 @@ if [[ -z $HOME ]]; then HOME=$(ls -d ~); fi
 # Copy CSS to Slack Folder
 sudo cp -af dark-theme.css "$THEME_FILEPATH"
 
+# if we have a custom file, append to the end.
+if [[ -f custom-dark-theme.css ]]; then
+  echo "Adding custom css"
+  cat custom-dark-theme.css >> "$THEME_FILEPATH"
```

Comment:
This will add custom styles every time the script is run, wont it?

Is it possible to create a temporary file (_by composing custom and main file_), which will be used as a theme file and remove it after the script is run?

Reply:
Yes, but Im also overwriting the theme file with the original one from the repo on line 40 to avoid any extra duplication, so it's kind of the same thing.